File: src/main/java/com/ByTech/ByTech/Fabricante/repositories/IEnterpriseRepository.java
package com.ByTech.ByTech.Fabricante.repositories;

import com.ByTech.ByTech.Fabricante.models.EnterpriseModel;
import org.springframework.data.jpa.repository.JpaRepository;

public interface IEnterpriseRepository extends JpaRepository<EnterpriseModel,Long> {
}

File: Movil/backend/ByTech_movil/src/main/kotlin/com/example/ByTech_movil/Fabricante/repositories/IEnterpriseRepository.kt
package com.example.ByTech_movil.Fabricante.repositories

import com.example.ByTech_movil.Fabricante.models.EnterpriseModel
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface IEnterpriseRepository : JpaRepository<EnterpriseModel, Long> {
    fun existsByNombre(nombre: String): Boolean
}
